src/exports/entrypoints/README.md

```
@@ -0,0 +1,46 @@
+1. This wouldn't include transition handlers. Those can't simply be passed through from the main thread, they need to be included in the worker code itself
+2. Despite workers being a standardised feature, the way build systems deal with them is not. Some won't allow for this kind of implementation to work well.
+
+   In particular, Vite has this problem. You can use two types of syntax to import workers:
+
+   ```ts
+   import worker from './something?worker'
+   ```
+
+   ```ts
+   const worker = new Worker(new URL('./something', import.meta.url), {
+   	type: 'module'
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 4

(MD010, no-hard-tabs)